### PR TITLE
Basket functionality

### PR DESCRIPTION
**Pull request by jcolairo**
- setting up to show products on the page by category

- got users to show on page

- able to add products to an order

### CODE REVIEW

File: frontend/js/directives/resize.directive.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ function resize () {
 
       function  checkScreen () {
         if(window.innerWidth < 780){
-          angular.element(element)         .parent().css('display', 'none');       
+          angular.element(element)         .parent().css('display', 'none');
```
Comment (CrocoNiall) [inline frontend/js/directives/resize.directive.js]: Whats with the space?

File: frontend/js/directives/resize.directive.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ function resize () {
...
 
           // angular.element(element).remove();
```
Comment (CrocoNiall) [inline frontend/js/directives/resize.directive.js]: Comment?

File: api/middleware/userPopulator.js [hunk 3]
```diff
@@ -7,38 +7,36 @@ module.exports  = function (req, res, next) {
...
+  };
   var ensureUserExists = function (user) {
-    
-      getUser(user.uid).then(function () {
-        next();
-      }).catch(function () {
-        createUser(user);
-      })
-  } 
+    // console.log(user)
```
Comment (CrocoNiall) [inline api/middleware/userPopulator.js]: remove log

File: frontend/js/directives/resize.directive.js (hunk 1 above)
Comment (jcolairo) [reply frontend/js/directives/resize.directive.js]: @morkett

**Pull request merged**